components/application-operator/charts/application/templates/role-binding.yaml

```diff
@@ -102,6 +102,7 @@ roleRef:
   kind: ClusterRole
   name: {{ .Release.Name }}-application-gateway
 {{- end }}
+{{- if not .Values.global.centralApplicationConnectivityValidatorEnabled }}
```

Comment:
I think the condition should go to the top. Otherwise for standalone mode unused stuff will be created.

Reply:
Same as above. PR for central validator and not central gateway ?